### DISCUSSION TITLE
Platform specific configuration

### DISCUSSION THREAD

**Discussion by chinise** [wezterm/wezterm#4342] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

I want to use my config file on multiple oses (I use dual boot).

Is there any way of checking os from the lua config file?

I looked though docs and didn't find anything useful so any help will be appreciated.

**Comment by tcoopman** [comment, marked as answer]
See here in the docs: https://wezfurlong.org/wezterm/config/lua/wezterm/target_triple.html
